src/pages/Login/Login.js

```diff
@@ -0,0 +1,75 @@
+import { PropTypes } from "prop-types";
+import { useState } from "react";
+import api from "../../api/api";
+import SignUp from "./SignUp";
+import { FetchState } from "../../hooks";
+
+const Login = ({ dispatch }) => {
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+  const [register, setRegister] = useState(false);
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+    dispatch({ type: FetchState.FETCH_INIT });
+    try {
+      await api.createSession(email, password);
+      const data = await api.getAccount();
+      dispatch({ type: FetchState.FETCH_SUCCESS, payload: data });
+    } catch (e) {
+      dispatch({ type: FetchState.FETCH_FAILURE });
+    }
+  };
+
+  return register ? (
+    <SignUp setRegister={setRegister} dispatch={dispatch} />
+  ) : (
+    <>
```

Comment:
You can skip the `<>` and use the `<section>`  since this only one-level element and not multiple elements. I'd export this huge block to another component, would be nicer, no?

Reply:
Yeap I agree with you 👍